{
    "client_datetime": "2024-08-12 17:13:46",
    "server_datetime": "2024-08-12 20:13:46",
    "data": [
        [
            "2024-08-12 17:13:17",
            "A123EE0",
            "erro_outros",
            "",
            "",
            "send_request",
            "",
            "maximum number of attempts to connect to server exceeded"
        ],
        [
            "2024-08-12 17:13:17",
            "A123EE0",
            "erro_outros",
            "",
            "",
            "request_datetime_server",
            "<class -UnboundLocalError->",
            "cannot access local variable -response- where it is not associated with a value"
        ]
    ]
}